cosmos/dbt/parser/project.py

```diff
@@ -274,7 +274,7 @@ class LegacyDbtProject:
 
     operator_args: Dict[str, Any] = field(default_factory=dict)
 
-    def __post_init__(self) -> None:
+    def __post_init__(self) -> None:  # noqa: C901
```

Comment:
@joppevos Perhaps, as opposed to skipping the complexity check here, we could use the `or` strategy as well., e.g:
```
self.dbt_snapshots_dir = self.dbt_snapshots_dir or "snapshots"
```
What are your thoughts?

Reply:
Yes lets do that!